### PR TITLE
add section on how to change default package manager.

### PR DESCRIPTION
**Pull request by dantereve**
## Description

Changing the default package manager used by `gatsby-cli`can be tricky for newcomers as there is no CLI command for this. I've added some explanation to help those who want to change it by modifying the configstore file.

### CODE REVIEW

**Comment by marcysutton** [comment]
Is there an issue related to this? It would be good to explain why someone would need or want to do this, and what the associated risks are (e.g. having to delete node_modules and .lock files, and reinstall).

**Comment by dantereve** [comment]
I've seen at least one issue related to this quite recently: #17094.

My PR was meant to explain how to change the default package manager for new projects only when using `gatsby new`. I'm going to modify my PR to be clearer on that matter and settle this Prettier issue :).

But it might be interesting to add some more details somewhere in the doc on how to change the package manager of a project in progress. Even if it looks like an edge case, it could be useful for developing a gatsby theme as we recommend yarn workspaces. Although this has nothing to do with the CLI.

File: docs/docs/gatsby-cli.md [hunk 1]
```diff
@@ -207,3 +207,30 @@ For more information, check out the [Gatsby REPL documentation](/docs/gatsby-rep
...
+
+```json:title=config.json
+{
+  "cli": {
+    "packageManager": "yarn"
+  }
+}
+```
+
+Edit your `packageManager` value, save and you’re good to go for your next project using `gastby new`.
```
Comment (marcysutton) [inline docs/docs/gatsby-cli.md]:
```suggestion
Edit your `packageManager` value, save and you’re good to go for your next project using `gatsby new`.
```

Comment (dantereve) [reply docs/docs/gatsby-cli.md]: Thank you for this, if I can't spell Gatsby I'm doomed!

**Pull request merged**